app/javascript/components/patient/close_contacts/CloseContactTable.js

```diff
@@ -0,0 +1,476 @@
+      }
+      axios.defaults.headers.common['X-CSRF-Token'] = this.props.authenticity_token;
+      axios
+        .post(url, {
+          patient_id: this.props.patient.id,
+          first_name: ccData.first_name || '',
+          last_name: ccData.last_name || '',
+          primary_telephone: ccData.primary_telephone ? phoneNumberToE164Format(ccData.primary_telephone) : '',
+          email: ccData.email || '',
+          last_date_of_exposure: ccData.last_date_of_exposure || null,
+          assigned_user: ccData.assigned_user || null,
+          notes: ccData.notes || '',
+          enrolled_id: ccData.enrolled_id || null,
+          contact_attempts: ccData.contact_attempts || 0,
+        })
+        .then(() => {
+          location.reload();
+        })
+        .catch(error => {
+          this.setState({ isLoading: false });
+          reportError(error);
+        });
+    });
+  };
+
+  /**
+   * Creates the action button & dropdown for each row in the table.
+   * @param {Object} data - Data about the cell this filter is called on.
+   */
+  renderActionsDropdown = data => {
+    const rowIndex = data.rowIndex;
+    const rowData = data.rowData;
+    // Set the direction to be "up" when there are not enough rows in the table to have space for the dropdown.
+    // The table custom class handles the rest.
+    // NOTE: If this dropdown increases in height, the custom table class passed to CustomTable will need to be updated.
+    const direction = this.state.table.rowData && this.state.table.rowData.length > 4 ? null : 'up';
+    return (
+      <Dropdown drop={direction}>
+        <Dropdown.Toggle
+          id={`close-contact-action-button-${rowData.id}`}
+          size="sm"
+          variant="primary"
+          aria-label={`Close Contact Action Button for Close Contact ${rowData.id}`}>
+          <i className="fas fa-cogs fw" />
+        </Dropdown.Toggle>
+        <Dropdown.Menu drop={direction}>
+          <Dropdown.Item className="px-4" onClick={() => this.toggleEditModal(rowIndex)}>
+            <i className="fas fa-edit fa-fw" />
+            <span className="ml-2"> Edit</span>
+          </Dropdown.Item>
+          <Dropdown.Item className="px-4" onClick={() => this.handleContactAttempt(rowIndex)}>
+            <i className="fas fa-phone fa-flip-horizontal" />
+            <span className="ml-2">Contact Attempt</span>
+          </Dropdown.Item>
+          {rowData.enrolled_id && (
+            <Dropdown.Item className="px-4" onClick={() => (location.href = patientHref(rowData.enrolled_id, this.props.workflow))}>
+              <i className="fas fa-search" />
+              <span className="ml-2">View Record</span>
+            </Dropdown.Item>
+          )}
+          {!rowData.enrolled_id && this.props.can_enroll_close_contacts && (
+            <Dropdown.Item className="px-4" onClick={() => (location.href = `${window.BASE_PATH}/patients/new${navQueryParam(this.props.workflow, true)}`)}>
```

Comment:
Looking at the [line that was originally removed](https://github.com/SaraAlert/SaraAlert/blob/ff02bb2fd8c6067cc5b2eed53f6e135304f3b41e/app/javascript/components/patient/close_contact/CloseContact.js#L364), we also need to pass the `cc` query param apparently. This is necessary so it auto-populates the data that's already been filled in for the close contact, like name.

Please put the cc param before the nav param. You'll also then need to pass `false` instead of `true` to `navQueryParam`.

Reply:
Ahh good catch

Reply:
fixed!